dead_songs/settings.py

```diff
@@ -167,7 +167,7 @@
 
 STATIC_ROOT = os.path.join(BASE_DIR, 'staticfiles')
 
-STATIC_URL = '/static/'
+STATIC_URL = '/neighborhood-development/static/'
```

Comment:
Need to replace this URL path with whatever default route is used for the exemplar app.